admin_guide/managing_networking.adoc

```diff
@@ -420,50 +421,82 @@ $ oc create -f <replication_controller>.json
 oc describe rc <replication_controller>
 ----
 
-[[multicast]]
+[[admin-guide-networking-multicast]]
 == Enabling Multicast
 
-Normally, multicast traffic between pods is disabled in {product-title}, but you
-can enable it on a per-project basis by setting an annotation on the project's
-corresponding NetNamespace object:
-
+[IMPORTANT]
+====
+Enabling multicast networking is a Technology Preview only. Technology
+Preview features are not supported with Red Hat production service level
+agreements (SLAs), might not be functionally complete, and Red Hat does not
+recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, 
+see https://access.redhat.com/support/offerings/techpreview/. 
 ====
+
+Multicast traffic between {product-title} pods is disabled by default. Enable it
+on a per-project basis by setting an annotation on the project's corresponding
+NetNamespace object:
+
 ----
-# oc annotate netnamespace <namespace_name> \
+# oc annotate netnamespace <namespace> \
     netnamespace.network.openshift.io/multicast-enabled=true
 ----
-====
-
-This is a Tech Preview feature and the details may change in the future.
 
-You can later disable multicast again by removing the annotation:
+Disable multicast by removing the annotation:
 
-====
 ----
-# oc annotate netnamespace <namespace_name> \
+# oc annotate netnamespace <namespace> \
     netnamespace.network.openshift.io/multicast-enabled-
 ----
+
+If you have
+xref:../admin_guide/managing_networking.adoc#joining-project-networks[joined
+networks together], you will need to enable multicast in each of the projects'
+NetNamespaces in order for it to take effect in any of the projects. To enable multicast in the `default` project, you must also enable
+it in all other projects that have been
+xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made
+global]. 
+
+[NOTE]
+====
+Multicast global projects are not "global", but instead communicate with only
+other global projects via multicast, not with all projects in the cluster, as is
+the case with unicast.
 ====
 
-If you have <<joining-project-networks,joined networks together>> with
-`oadm pod-network` you will need to enable multicast in each of the projects'
-NetNamespaces in order for it to take effect in any of the projects. You cannot
-enable multicast in the "default" project or in any other project that has been
-<<making-project-networks-global,made global>>, nor can you send multicast
-packets into or out of global projects.
+[[admin-guide-networking-networkpolicy]]
+== Enabling NetworkPolicy
 
-[[networkpolicy]]
-== NetworkPolicy
+[IMPORTANT]
+====
+Enabling the Kubernetes NetworkPolicy is a Technology Preview only. Technology
```

Comment:
only a Technology Preview feature.

Same as above. Consider linking to https://access.redhat.com/support/offerings/techpreview/ in this first reference instead of stating "For more information..."